### PR TITLE
editorial: add CR exit criteria

### PR DESCRIPTION
**Pull request by marcoscaceres**
* Addresses  the "must document how adequate implementation experience will be demonstrated" for CR.

***
[Preview](https://s3.amazonaws.com/pr-preview/w3c/payment-method-id/advancement.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/payment-method-id/253f9e2...16aa920.html)

### CODE REVIEW

**Comment by marcoscaceres** [comment]
@ianbjacobs ping.

**Comment by marcoscaceres** [comment]
This now matches what is in the Payment Request API's  conformance section.

File: index.html [hunk 1]
```diff
@@ -187,6 +187,17 @@ <h3 id="registry">
         </dd>
       </dl>
     </section>
+    <section id="conformance">
+      <p>
+        The working group will demonstrate implementation experience by
+        creating a <a href="https://w3c-test.org/payment-method-id/">test
+        suite</a> and having at least two independent implementations pass each
+        mandatory test (i.e., each test the corresponds to a MUST requirements
```
Comment (sideshowbarker) [inline index.html]: mandatory test (i.e., each test the corresponds to a MUST requirements

_requirement_ (without the _s_)

Comment (marcoscaceres) [reply index.html]: Thanks, will fix in the other spec too.

**Review by ianbjacobs — COMMENTED** [review]
Hi all,

I would like to again request that this information be m moved out of the conformance section and into the status section of the document. 

This information is not related to how user agents conform to the specification, but rather how W3C will demonstrate interop.

**Comment by marcoscaceres** [comment]
Heh, I had it in tge SOtD, but moved it because PR API had it in the wrong place.

**Pull request merged**